packages/rum/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -83,6 +88,11 @@ export function trackViews(location: Location, lifeCycle: LifeCycle) {
   )
 
   return {
+    setCustomTiming(name: string, inInitialView = false, time = performance.now()) {
```

Comment:
`addTiming` is a bit misleading since we set a new value for the timing if already exists. Wouldn't it be better to name it `setTiming`?

Reply:
Do you mean for the public API?
it is maybe a bit late for that since afaik it is implemented on the mobile side, you can launch a discussion on that if you think it is better

Reply:
if it's already on the public API of mobile SDK, it makes sense to use the same name here. I will replace all `setCustomTiming` with `addTiming`.